db/schema.rb

```diff
@@ -624,6 +624,7 @@
     t.datetime "created_at", precision: nil, null: false
     t.datetime "updated_at", precision: nil, null: false
     t.boolean "platform_authenticator"
+    t.string "transports", array: true
```

Comment:
Alternatives: Seems like this could be a nice use-case for an array enum, but that's not a thing. Bitmask is another option, based on some discussion I've seen, but may be unnecessary added complexity.